java/apirag-udf/src/test/java/com/datasqrl/ai/function/TestFunctionConversion.java

```diff
@@ -0,0 +1,65 @@
+package com.datasqrl.ai.function;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.datasqrl.ai.backend.FunctionType;
+import com.datasqrl.ai.backend.RuntimeFunctionDefinition;
+import com.datasqrl.ai.function.builtin.BuiltinFunctions;
+import com.datasqrl.ai.function.builtin.CurrentTime;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.time.Instant;
+import java.util.Set;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.reflections.Reflections;
+
+public class TestFunctionConversion {
```

Comment:
Shouldn't this be called `FunctionConversionTest`?

Reply:
Correct and fixed.